prow/apis/prowjobs/v1/types.go

```diff
@@ -474,7 +474,7 @@ func (u *UtilityImages) ApplyDefault(def *UtilityImages) *UtilityImages {
 		return u
 	}
 
-	merged := *u
+	merged := *u.DeepCopy()
```

Comment:
This isn't really necessary today as the `*UtilityImages` does not contain any pointers, but I added it to make sure it stays threadsafe should that ever change